core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/TelemetryConverter.java

```diff
@@ -367,10 +364,12 @@ private void setReportDeploymentLogs(TelemetryRequest request, Features features
         }
     }
 
-    private void setUseSharedAltusCredential(TelemetryRequest request, Features features) {
-        if (useSharedAltusCredential && request.getFeatures() != null && request.getFeatures().getUseSharedAltusCredential() != null) {
-            LOGGER.debug("Fill shared altus credential setting from telemetry feature request");
-            features.setUseSharedAltusCredential(request.getFeatures().getUseSharedAltusCredential());
+    private void setUseSharedAltusCredential(Features features) {
+        if (useSharedAltusCredential) {
+            LOGGER.debug("Fill shared altus credential setting as that is enabled globally");
+            FeatureSetting useSharedAltusCredentialFeature = new FeatureSetting();
+            useSharedAltusCredentialFeature.setEnabled(true);
+            features.setUseSharedAltusCredential(useSharedAltusCredentialFeature);
```

Comment:
this logic occurs three times in this PR, maybe it is worth to extract it into some static class and call it everywhere

Reply:
it's just a few steps, maybe in features object there should be a method to set this wrapper objects based on a boolean param and add a `@JsonIgnore` there. but that should be done to every occurrences of these feature settings, expect that this PR will be probably larger